api/app/Models/PoolCandidate.php

```diff
@@ -261,6 +261,20 @@ public static function scopeExpiryStatus(Builder $query, ?string $expiryStatus)
         return $query;
     }
 
+    public static function scopeSuspendedStatus(Builder $query, ?string $suspendedStatus)
```

Comment:
I'm a bit uncomfortable with how this overlaps with `scopeAvailable` but I'm not sure if that can be avoided. :shrug:

Reply:
🤷